manifests/charts/istio-control/istio-discovery/files/gateway-injection-template.yaml

```diff
@@ -13,9 +13,13 @@ metadata:
   }
 spec:
   securityContext:
+  {{- if .Values.global.securityContext }}
```

Comment:
I feel like "global.securityContext" is not a great name for this API.

its not really global, it only applies to gateways. Maybe we should have some way to indicate that

Reply:
ye, I'm struggling with the definition, but don't I have a better idea where to define this.

Does anyone have any idea?